boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.mk

```diff
@@ -2,22 +2,22 @@
 	
 .PHONY: staging-hack-csv-build
 staging-hack-csv-build: 
-	@${CONVENTION_DIR}/csv-generate/csv-generate.sh -o $(OPERATOR_NAME) -i $(OPERATOR_IMAGE) -c staging -H $(CURRENT_COMMIT) -n $(COMMIT_NUMBER) -g hack
+	@${CONVENTION_DIR}/csv-generate/csv-generate.sh -o $(OPERATOR_NAME) -i $(OPERATOR_IMAGE_URI) -c staging -H $(CURRENT_COMMIT) -n $(COMMIT_NUMBER) -g hack
```

Comment:
In https://github.com/openshift/boilerplate/pull/134 I reverted this and added a param for `-V $(OPERATOR_VERSION)` to facilitate computing the by-digest URI -- see https://github.com/openshift/boilerplate/pull/134/files#diff-424ddf34b68cef0633f4520f756a919d70a24880b5ab931d3cd85d17f8659b5fR38-R58